### PR TITLE
Add telephone number react page

### PR DESCRIPTION
**Pull request by frederike-ramin**
# Short Description
- This adds the complete react page for the telephone number.
- I had to merge #424 to use the input field, so please review that first. (otherwise: This is just about the files `webapp/client/src/pages/TelephoneNumberPage.js`, `webapp/client/src/pages/TelephoneNumberPage.test.js`, `webapp/client/src/stories/TelephoneNumberPage.stories.js`, `webapp/client/src/components/OptionalHint.js` and `webapp/client/src/lib/translations.js`)
- This does not wire it up yet with the webapp or erica.

# Changes
- Add margin to "optional" span
- Add react page
- Add stories
- Add tests

# Feedback
- More stories? More tests?

### CODE REVIEW

File: webapp/client/src/pages/TelephoneNumberPage.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}) {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <StepHeaderButtons url={prevUrl} />
+      <FormHeader {...stepHeader} />
+      <StepForm {...form}>
+        <FormFieldTextInput
+          autofocus
```
Comment (JulianRoesner) [inline webapp/client/src/pages/TelephoneNumberPage.js:24]: Do we really want to set autofocus here? Is that still part of the design spec?

**Review by JulianRoesner — APPROVED** [review]
LGTM. Just some small comments

File: webapp/client/src/components/OptionalHint.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { useTranslation } from "react-i18next";
 import styled from "styled-components";
 
 const Hint = styled.span`
+  margin-left: var(--spacing-01);
```
[the remark this answers is not in the document]
Comment (frederike-ramin) [reply webapp/client/src/components/OptionalHint.js:6]: Before, the "optional" text was glued to the actual label without spacing. We just never used it before that's why it never came up.

File: webapp/client/src/pages/TelephoneNumberPage.js (hunk 1 above)
Comment (frederike-ramin) [reply webapp/client/src/pages/TelephoneNumberPage.js:24]: Good question.

Comment (frederike-ramin) [reply webapp/client/src/pages/TelephoneNumberPage.js]: Nadine said no^^

**Pull request merged**